lib/entities/workflow-definition.ts

```diff
@@ -14,14 +14,32 @@ import { wrapCollection } from '../common-utils'
 import enhanceWithMethods from '../enhance-with-methods'
 import { ActionType } from './role'
 
-/* Workflow Step */
+/* Workflow Step Action */
+export enum WorkflowStepActionType {
+  App = 'app',
+  Email = 'email',
+  Task = 'task',
+}
+export interface WorkflowStepAction<T = Record<string, any>> {
+  appId?: string
+  appActionId?: string
+  type: WorkflowStepActionType
+  configuration?: T
+}
+
+type EmailActionRecipient = string | Link<'User'> | Link<'Team'>
 
-export type WorkflowStepAction = {
-  action: Link<'AppAction'>
+export type WorkflowStepEmailActionConfiguration = {
+  recipients: EmailActionRecipient[]
+}
+
+export type WorkflowStepTaskActionConfiguration = {
```

Comment:
Let's also add `WorkflowStepAppActionConfiguration` with an optional JSON body and headers. Basically, type the login presented here - https://github.com/contentful/workflow-management-api/blob/f2b7351152813b72da82b1548090e7a6d2ef9844/src/domain/workflow-definition/validations.ts#L217

Reply:
done